modules/cpdos/hhcn.py

```diff
@@ -2,18 +2,21 @@
 # -*- coding: utf-8 -*-
 
 """
-Attempts to find Cache Poisoning with Host Header Case Normalization (HHCN)  
+Attempts to find Cache Poisoning with Host Header Case Normalization (HHCN)
 https://youst.in/posts/cache-key-normalization-denial-of-service/
 """
 
-from ..utils import *
+from modules.utils import random, requests, urlparse, configure_logger
+
+logger = configure_logger(__name__)
 
 VULN_NAME = "Host Header Case Normalization"
 
-def HHCN(url, s, authent):
-    behavior = False
+CONTENT_DELTA_RANGE = 50
+
 
-    # replace min char by maj char in the domain
+def random_domain_capitalization(url):
+    """Randomly capitalize characters from the url domain"""
```

Comment:
This function does its job, but I hesitated to have it done with a simple list comprehension.


@c0dejump Also, it's randomly picking one index, but would it be worth to :
- have more than one character capitalized ?
- have several capitalization possible to enumerate on ?